packages/currency/src/aggregators/fantom.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Idea: also move it to a `/chainlink/` subdirectory, under `/aggregators/`

Reply:
we will do that when we have other providers ;)